packages/itwinui-react/src/core/Dialog/DialogContext.tsx

```diff
@@ -70,6 +70,10 @@ export type DialogContextProps = {
    * Dialog root ref. For internal use.
    */
   dialogRootRef?: React.RefObject<HTMLDivElement>;
+  /**
+   * Determines the positioning of Dialog on page.
+   */
```

Comment:
Perhaps add @default 'top-left' here for clarity.

Reply:
There is no default placement. 
Default placement is undefined and the dialog is placed in the center of the page.